client/src/js/references/components/Detail/Targets/__test__/Item.test.js

```diff
@@ -0,0 +1,15 @@
+import { TargetItem, mapStateToProps, mapDispatchToProps } from "../Item";
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [mapDispatchToProps not found in '../Item'](https://app.codacy.com/manual/virtool/virtool/pullRequest?prid=4894825)